app/models/ride.rb

```diff
@@ -8,10 +8,37 @@ class Ride < ApplicationRecord
   belongs_to :ride_zone
   has_one :conversation
 
+  scope :completed, -> { where(status: :complete)}
+
   validates :voter, presence: true
 
   after_create :notify_creation
   around_save :notify_update
+  before_save :close_conversation
```

Comment:
Rename to close_conversation_if_ride_complete or the like? At first I thought it simply closed the conversation, without a condition.